.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: 18
+          node-version: 20
```

Comment:
This bump is causing CI to fail for Vercel. Noted your comment on RR7 dependencies, though. Need to probe this a bit further.

Reply:
I could replicate the issue locally, moving to v22 (LTS) works so will try on CI once I've sorted a monorepo issue.